R/importQIIME2.R

```diff
@@ -68,57 +80,63 @@
 #' \url{https://qiime2.org}
 #'
 #' @examples
-#' featureTableFile <- system.file("extdata", "table.qza", package = "mia")
-#' taxonomyTableFile <- system.file("extdata", "taxonomy.qza", package = "mia")
-#' sampleMetaFile <- system.file("extdata", "sample-metadata.tsv", package = "mia")
-#' phyTreeFile <- system.file("extdata", "tree.qza", package = "mia")
-#' refSeqFile <- system.file("extdata", "refseq.qza", package = "mia")
+#' assay.file <- system.file("extdata", "table.qza", package = "mia")
+#' row.file <- system.file("extdata", "taxonomy.qza", package = "mia")
+#' col.file <- system.file("extdata", "sample-metadata.tsv", package = "mia")
+#' phy.tree.file <- system.file("extdata", "tree.qza", package = "mia")
+#' refseq.file <- system.file("extdata", "refseq.qza", package = "mia")
 #' tse <- importQIIME2(
-#'   featureTableFile = featureTableFile,
-#'   taxonomyTableFile = taxonomyTableFile,
-#'   sampleMetaFile = sampleMetaFile,
-#'   refSeqFile = refSeqFile,
-#'   phyTreeFile = phyTreeFile
+#'   assay.file = assay.file,
+#'   row.file = row.file,
+#'   col.file = col.file,
+#'   refseq.file = refseq.file,
+#'   phy.tree.file = phy.tree.file
 #' )
 #'
 #' tse
 
 #' @importFrom S4Vectors make_zero_col_DFrame
 importQIIME2 <- function(featureTableFile,
+                            row.file = taxonomyTableFile,
                             taxonomyTableFile = NULL,
+                            col.file = sampleMetaFile,
                             sampleMetaFile = NULL,
+                            as.refseq = featureNamesAsRefSeq,
                             featureNamesAsRefSeq = TRUE,
+                            refseq.file = refSeqFile,
                             refSeqFile = NULL,
+                            phy.tree.file = phyTreeFile,
                             phyTreeFile = NULL,
+                            assay.file = featureTableFile,
```

Comment:
Also here the order should be modified. assay.file must be the first parameter

Reply:
ACK